lib/mods/utils.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * Gets the Collection of CallExpressions with callee name console.
+ *
+ * @param {any} j - the jscodemod api
+ * @param {Collection} root - the root Collection
+ * @return {Collection} - a Collection of console statements
+ *
+ * @example
+ * getConsoleStatments(j, root).filter(...);
+ */
+
+const getConsoleStatments = (j, root) => {
```

Comment:
It's a style choice but you could also do `exports.getConsoleStatements = (j, root) => { ... }` instead of separately assigning `module.exports`.

Reply:
I think I'm going to go the route of separately defining all the functions before exporting them, and I like the format where they all export as an object:

```js
module.exports = {
  getConsoleStatements,
  ...
}
```